projects/abp-libraries/lib-post/src/lib/lib-post.component.scss

```diff
@@ -1,15 +1,16 @@
 @import 'styles/theme';
 
 .container {
+
   @extend %d-grid;
   width: 100%;
-  height: 10%;
+  height: 6%;
   background-color: #ffffff;
```

Comment:
create color variable in file theme.scss

Reply:
aproove